packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -86,15 +103,16 @@ class BpkDatepicker extends Component {
       initiallyFocusedDate,
       renderTarget,
       valid,
+      // onDateSelect, onClose and isOpen are decomposed here so they don't end
+      // up in the rest object
+      onDateSelect,
+      onClose,
```

Comment:
We shouldn't declare variables unless we plan to use them. I think the `delete` syntax was more appropriate.

Reply:
Cool, I like to avoid delete statements but I don't have a strong preference. I reverted to the way it was before